src/ZeekList.h

```diff
@@ -20,7 +20,6 @@ class Type;
 
 using ValPList = PList<Val>;
 using ExprPList = PList<detail::Expr>;
-using IDPList = PList<detail::ID>;
```

Comment:
I'm wavering on whether we can move this without a deprecation cycle. I agree that it's based on something that's in the detail namespace and I already checked that no external plugins use it. Anything using it would be broken badly because the API completely changed underneath.

Reply:
(well you know my view ...)